pandas/io/json/json.py

```diff
@@ -417,8 +420,12 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=None,
 
     if orient == 'table' and dtype:
         raise ValueError("cannot pass both dtype and orient='table'")
+    if orient == 'table' and convert_axes:
+        raise ValueError("cannot pass both convert_axes and orient='table'")
 
     dtype = orient != 'table' if dtype is None else dtype
+    if convert_axes is None:
+        convert_axes = orient != 'table'
```

Comment:
Can we just assign the literal `True` here instead of using the expression? I think it will help readability

Reply:
@WillAyd I do not understand what you mean: I cannot just assign the literal `True` here because the assignment must be:
- `True` if orient is different from 'table'
- `False` if orient equals 'table'

Reply:
Yea good point - thanks for confirming

Reply:
OK, I see: if orient is 'table', it can be left as None (no need to set it False), because the code only check `if self.convert_axes:`.